### DISCUSSION TITLE
⁉️ Ask your questions to Microsoft experts

### DISCUSSION THREAD

**Discussion by aycabas** [microsoft/hack-together-teams#3] [Q&A]
Please use discussions to ask your questions to Microsoft experts. We are regularly monitoring this discussion to answer all of your questions.

**Comment by IlonaHamanova** [comment]
Hello. I am teacher at technical high school, teaching computer sience and programing. I would like to try to develop apps for MS Teams with some of my students. But the developer tenant is only buisness and we want to develop for Teams for education. Our school has TfE, but we cant develop on it because of safety isues. So we need some other tenant of TfE for development. How can we get it? Thans for answer. Tried to ask on many places, but still have no answer. :-/

**Reply by aycabas** [reply]
Hi @IlonaHamanova, for development and testing purposes, we recommend you to use Microsoft 365 Developer Program tenants, you can create for free: https://aka.ms/m365developers 

Microsoft 365 Developer Program gives you a free tenant that includes pre-populated users and admin access to the entire tenant.

It's recommended to publish it on your production (Teams), after the development, debugging and testing phases.

**Reply by IlonaHamanova** [reply]
Hi @aycabas , I know, we have this developer tenent. But we want to make apps for teaching/learning and we want to work with assignments and other stuf in Teams for Education. And the developer tenant is business type. Or am I looging wrong and there is some possibility to download developer tenant which is type for education?

**Comment by KamilSiebyla** [comment]
Hi, 

could you confirm, that I am registered? I am back today from vacations, so got small amount of time to do something, but I will try to submit some MVP till the deadline :-D 

/Kamil

**Reply by waldekmastykarz** [reply]
Yes, you are registered 👍

**Reply by KamilSiebyla** [reply]
Thanks for  confirmation :) 

/Kamil
